15:01:01.063 INFO Test execution started: order_creation
15:01:01.063 INFO Browser: chrome
15:01:01.064 INFO Environment: test_env
15:01:01.064 INFO Using data:
    order_type: Real Property Recordings
    document_type: ABANDONMENT
    num_of_pages: 1
    section: GRANTEE
    first_name: fname
    section2: GRANTEE
    parceId: 1
    url: http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=
15:01:04.297 INFO Navigate to: 'http://User1:Automation_1_3@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer='
15:01:12.273 INFO Click addNewOrderButton
15:01:13.781 INFO Waiting for 3 seconds
15:01:16.792 INFO Click orderTypeDropdown
15:01:16.875 INFO Click //select[@id='orderTypeId']/option[text()='Real Property Recordings']
15:01:19.175 INFO Click documentTypeDropdown
15:01:19.291 INFO Click //select[@id='Order_OrderItems[0]_Document_DocumentTypeId']/option[text()='ABANDONMENT']
15:01:19.394 INFO Write '1' in element numberOfPages
15:01:19.504 INFO Click partiesTab
15:01:34.902 ERROR ElementNotFound: Element /input[@placeholder='First Name'] not found using selector xpath:'/input[@placeholder='First Name']'
Traceback (most recent call last):
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\test_runner\test_runner.py", line 173, in run_test
    self.test_module.test(execution.data)
  File "C:\Users\alexandr.may\testdir\projects\some_project\tests\order_creation.py", line 22, in test
    a.send_keys(new_order_page_form.firstNameInput(data.section), data.first_name)
  File "C:\Users\alexandr.may\testdir\projects\some_project\pages\new_order_page_form.py", line 9, in firstNameInput
    return partiesSection('GRANTEE').find(xpath="/input[@placeholder='First Name']")
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\extended_webelement.py", line 68, in find
    return _find(self, **kwargs)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\common.py", line 122, in _find
    element_name, timeout, wait_displayed)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\common.py", line 47, in _find_webelement
    .format(element_name, selector_type, selector_value))
golem.core.exceptions.ElementNotFound: Element /input[@placeholder='First Name'] not found using selector xpath:'/input[@placeholder='First Name']'

15:01:37.438 INFO Test Result: CODE ERROR
